### DISCUSSION TITLE
中英文摘要之间空页问题

### DISCUSSION THREAD

**Discussion by Kelipapa** [obster-y/XJTU-thesis#78] [Ideas]
当中文摘要1页，英文摘要2页的时候：
默认排版是《中文》《空白页》《英文》《英文》
打印店要求中英文摘要之间不空页，即：
《中文》《英文》《英文》《空白页》

因此修改了XJTU-thesis.cls
![image](https://user-images.githubusercontent.com/106446236/170857270-c3a08efc-1e67-4fe7-bf7c-a51b824b08e9.png)
将此处的\cleardoublepage注释掉
供参考

**Comment by obster-y** [comment]
同样地，去搜 issue，关键词「奇数页」，早已提到过

**Reply by obster-y** [reply]
在那些 issue 里，同样说明了不会做修改的理由
